### PR TITLE
Payment and Token Transfer Errors

### PR DESCRIPTION
**Pull request by godith0**
Screen Shot 2021-03-15 at 3 53 28 AM

### CODE REVIEW

**Comment by Juxsta** [comment]
It looks good to me!

File: src/views/Portfolio/views/Balance.vue [hunk 1]
```diff
@@ -528,36 +541,52 @@ export default {
...
-                sender_id: $senderId
-                recipient_email: $recipientEmail
-              }
+      loading.value = true;
+      try {
+        if (transferQuantity.value > tokens.value.length) {
+          throw new Error('You tried to send more tokens than your account holds.');
+        }
+        if (transferQuantity.value > 100) {
+          throw new Error('Please transfer your tokens in smaller batches (< 100 per batch).');
```
Comment (godith0) [inline src/views/Portfolio/views/Balance.vue:553]: @dericklee510 let me know if you want to change any of these error messages. Feel free to hop in and edit them as well.

Comment (dericklee510) [reply src/views/Portfolio/views/Balance.vue:553]: Will diggity do

**Review by dericklee510 — APPROVED** [review]
lets do dis

**Pull request merged**